projects/fsastorefrontlib/src/core/my-account/facade/quote.service.ts

```diff
@@ -133,11 +139,12 @@ export class QuoteService {
   protected loadChooseCoverForm(
     insuranceQuote: any,
     categoryCode: string
-  ): Observable<void> {
-    if (insuranceQuote && insuranceQuote.quoteDetails) {
-      const dataId = insuranceQuote.quoteDetails.formId;
+  ): Observable<any> {
+    const dataId = insuranceQuote.quoteDetails?.formId;
+    if (insuranceQuote && insuranceQuote.quoteDetails && dataId) {
```

Comment:
I would suggest adding optional operator after `insuranceQuote` and then there is no need for extra checks:
```
   const dataId = insuranceQuote?.quoteDetails?.formId;
    if (dataId) {
```